test/distributed/_composable/test_fsdp_logging.py

```diff
@@ -0,0 +1,58 @@
+# Owner(s): ["module: fsdp"]
+import functools
+import os
+import unittest.mock
+
+import torch.distributed as dist
+from torch.testing._internal.common_distributed import skip_if_lt_x_gpu
+from torch.testing._internal.inductor_utils import HAS_CUDA
+from torch.testing._internal.logging_utils import LoggingTestCase
+
+requires_cuda = unittest.skipUnless(HAS_CUDA, "requires cuda")
+requires_distributed = functools.partial(
+    unittest.skipIf, not dist.is_available(), "requires distributed"
+)
+
+
+@skip_if_lt_x_gpu(2)
+class LoggingTests(LoggingTestCase):
+    @requires_distributed()
```

Comment:
For my understanding, is this based on an existing test?

Reply:
Yeah, that's based on the [test_distributed_rank_logging](https://github.com/pytorch/pytorch/blob/163847b1bb5cc36a0915a189b2dd4cfbbfaf9c49/test/dynamo/test_logging.py#L476C9-L476C38) at dynamo's test_logging.py